packages/cli/docs/cli.md

```diff
@@ -267,49 +267,19 @@ $ zapier describe
 
 ## env
 
-  > Read, write, and delete environment variables.
+> Gets Environment variables for a version.
 
-  **Usage:** `zapier env 1.0.0 CLIENT_SECRET 12345`
-
-  
-Manage the environment of your app so that `process.env` has the necessary variables, making it easy to match a local environment with a production environment via `CLIENT_SECRET=12345 zapier test`.
+**Usage**: `zapier env VERSION`
```

Comment:
The doc doesn't seem to fully reflect the command. Maybe change it to:

```
> Gets, sets, or unsets environment variables for a version.

**Usage**: `zapier env:COMMAND VERSION`
```

Would be nice if we have at least one example for `env:get`, `env:set`, and `env:unset`, each.

Reply:
good call! tweaked it so subcommands all show up properly in docs & help.